modules/ROOT/pages/lb-mapping-rules.adoc

```diff
@@ -8,165 +8,292 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
+
+The order of the rules determines the priority. The first rule in the list that matches is applied. 
+
+[NOTE]
+If you don't define any mapping rules, the DLB creates and uses a default mapping rule.
 
-IMPORTANT: Patterns in the application URI are not currently supported.
 
-A pattern is a string that defines a template for matching an input text. Whatever value is placed within curly brackets (`{   }`) is treated as a variable.
-Variable names can contain only lowercase letters (a-z) and no other characters. Variable values can contain the characters `a-z0-1.&?-_`, but can't contain slashes.
+== Create Mapping Rules
+
+Mapping rules are attributes of the load balancer's SSL endpoint, which is identified by the certificate Common Name (CN).
+When you create mapping rules with:
+
+* Runtime Manager, you define mapping rules within a certificate definition. 
+* The CLI or CloudHub API, you must specify a certificate CN.
++
+If you omit the certificate name, the mappings are added to the default endpoint.
 
-Suppose you set up a DNS CNAME record from  `example.com` to `_lb-name_.lb.anypointdns.net`. You can then map `app.example.com` to a different deployed CloudHub Mule application name.
+=== Create Mapping Rules in Runtime Manager 
 
-For example, you can bind two hostnames for redirect:
+To create mapping rules in an SSL endpoint for a DLB in Runtime Manager:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Load Balancers* in the left menu.
+. Click a load balancer name, then click a certificate name.
+. In the *URL Mapping Rules* section, click *Add New Rule*:
++
+.The screenshot shows (*1*) the *Add New Rule* icon and (*2*) the fields required to create a mapping rule.
+image::cloudhub-dlb-mapping-rules.png[the *Add New Rule* icon and the fields required to create a mapping rule]
+. Complete the fields:
++
+* *Input Path* (`inputUri`)
++ 
+The URI that the client requests: for example, `/{app}/`.
++
+The input URI is appended to the host header of the load balancer.
+* *Target App* (`appName`)
++
+The name of the CloudHub application that processes the request: for example, `{app}-example`.
+* *Output Path* (`appURI`)
++
+The URI string to pass to the app: for example, `/`.
++
+The output path cannot contain patterns.
+* *Protocol* (`upstreamProtocol`)
++
+The protocol on which the application listens:
++
+--
+** `http` (port 8091)
+** `https` (port 8092)
+** `ws` WebSockets (port 8091)
+** `wss` Web Service Security (8092)
+
+By default, the load balancer listens to external requests on HTTPS and communicates internally with your worker over HTTP. If you configured your Mule application within the VPC to listen on HTTPS, make sure you set *Protocol* to `https` when you create the mapping rule list.
+--
+
+You can use either literal strings or patterns to define the input path. 
+For the output path, you can use only literal strings.
```

Comment:
I kept rereading this topic set, and discovered that it would be more clear to me if there were a concept text about pattern mapping vs. literal mapping before getting into the priority texts.